packages/fxa-auth-server/scripts/test-ci.sh

```diff
@@ -3,35 +3,42 @@
 DIR=$(dirname "$0")
 cd "$DIR/.."
 
-yarn workspaces focus fxa-auth-server
+# TODO: Shouldn't be needed... Validate that's true.
+# yarn workspaces focus fxa-auth-server
 
 export NODE_ENV=dev
 export CORS_ORIGIN="http://foo,http://bar"
 
-DEFAULT_ARGS="--require esbuild-register --recursive --timeout 5000 --exit --reporter mocha-junit-reporter"
+DEFAULT_ARGS="--require esbuild-register --recursive --timeout 5000 --exit "
+if [ "$TEST_TYPE" == 'unit' ]; then GREP_TESTS="--grep #integration --invert "; fi;
+if [ "$TEST_TYPE" == 'integration' ]; then GREP_TESTS="--grep #integration "; fi;
 
 node -r esbuild-register ./scripts/gen_keys.js
 node -r esbuild-register ./scripts/gen_vapid_keys.js
 node -r esbuild-register ./scripts/oauth_gen_keys.js
-../../_scripts/check-mysql.sh
-../../_scripts/check-url.sh localhost:9090
-node ../db-migrations/bin/patcher.mjs
-
-# Make sure we have a fresh version of l10n files... This is just a sanity check. It shouldn't be needed...
-# rm -rf public
-# yarn run postinstall
 
+echo 'Updating ftl files'
 # Migrate current strings
 yarn run merge-ftl
 yarn run merge-ftl:test
 
 # Process sass for rendering of email templates
+echo
 yarn run emails-scss
 
-TESTS=(local oauth remote scripts)
+if [ "$TEST_TYPE" == 'integration' ]; then
+  TESTS=(local oauth remote scripts);
```

Comment:
I think for auth-server, the integration tests should be the `oauth remote` and others are unit. I would probably consider any test suite that start the [test-server](https://github.com/mozilla/fxa/blob/1189bf7cafec66569f6183758be5c1d4263a21a5/packages/fxa-auth-server/test/test_server.js#L21) to be integration.